pycue/opencue/wrappers/host.py

```diff
@@ -403,6 +403,12 @@ def lockState(self):
         """
         return self.data.lock_state
 
+    def os(self):
+        """
+        :rtype: opencue.os
```

Comment:
I think the `rtype` here is just `str`, right?

Reply:
eh, yes, updated. thanks!